content/jobs/README.md

```diff
@@ -0,0 +1,46 @@
+# Job Post Instructions
+
+---
+
+Hey there! If you're a local-ish tech company looking to hire talent we at NewHaven.io would love to help out by putting some information about your organization and open role(s) on our website.
+
+Just clone this repository, add your information to [this page](https://github.com/newhavenio/newhavenio/tree/master/content/jobs/jobs.yaml), and submit a pull request.
+
+```markdown
+- company: "NewHaven.IO"
+  jobURL: "https://newhaven.io"
+  jobTitle: "Community Moderator"
+  location: "New Haven, CT // Remote"
+  tech: "Slack, MeetUp, GatsbyJS"
+  salary: "Volunteer"
+  description: "We're looking for help with some community moderation. As vaccines keep rolling out we're looking at ways we can begin to breathe life back into the group. We want to wade back into holding some events and we're going to need help keeping an eye our Slack team and Meetup page. Over time we can fold whoever takes this role into our organizing and leadership teams. If you've gotten some value from IO in the past we'd love your help in carrying it forward."
+```
+
+Add information on your role(s) as a new object(s) at the bottom of the file. Here's an example:
+
+```markdown
```

Comment:
Is this a duplication?

Reply:
Not sure what you mean.

Reply:
Wanted to make sure there was some information on how to add another job to the bottom of the file.

Reply:
I guess I'm not sure why we need two examples in the same file

Reply:
Removed and pushed.